tox.ini

```diff
@@ -5,12 +5,12 @@ envlist =
     pypy
 
 [testenv]
-commands = py.test -q tests.py
+commands = py.test
 deps = pytest
 setenv =
     PIP_DISABLE_PIP_VERSION_CHECK = 1
 
 [testenv:flake8]
 basepython = python3.4
 deps = flake8
-commands = flake8
+commands = flake8 --ignore=F401
```

Comment:
Why was skipping the F401 was necessary?

Reply:
Since __init__.py imports what is needed into the root namespace, but doesn't use anything itself, it detects unused variables. It seems other projects do this too:

https://github.com/requests/requests/blob/5dcbdb50fc9652cbeffeec22a21d1196e938bfca/Makefile#L14